tests/br_tls/certificates/ca.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This certificate expires in 5 years at `Feb 11 04:12:00 2025 GMT`. 

If you're going to hard-code the certs please make the expiry date longer (for comparison TiKV's test cert expires in 100 years at `Oct 31 21:12:44 2117 GMT`.)